stories/features/newTab/components/clock/style.ts

```diff
@@ -0,0 +1,57 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+const StyledClock = styled.div`
+  color: inherit;
+  box-sizing: border-box;
+  line-height: 1;
+  user-select: none;
+  display: flex;
+  cursor: default;
+  -webkit-font-smoothing: antialiased;
```

Comment:
Maybe more of a question than anything. Everything I've seen about font-smoothing always concludes with: "only use font-smooth overrides sparingly and as a last resort"

Reply:
ya I don't think there are any side-effects that would prevent us from using it. I'm open for discussion on this topic but I don't see any reason why not to use the way we are doing